### PR TITLE
feat: update module

### PR DESCRIPTION
**Pull request by appleboy**
Signed-off-by: Bo-Yi Wu <appleboy.tw@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,14 +6,14 @@ VERSION_PATCH  := 1
...
-PKGS    := $(shell go list ./... | grep -v /vendor)
+PKGS    := $(shell go list ./...)
 REPO    := github.com/guessi/kubectl-search
 VERSION := v$(VERSION_MAJOR).$(VERSION_MINOR).$(VERSION_PATCH)$(VERSION_SUFFIX)
 LDFLAGS := -s -w -X $(REPO)/cmd.appVersion=$(VERSION)
 
 export GO111MODULE=on
 
-default: build
+all: build
```
Comment (guessi) [inline Makefile]: conflict with the "all" defined at line 53

File: go.mod [hunk 2]
```diff
@@ -16,19 +16,21 @@ require (
...
 	go.opencensus.io v0.19.1 // indirect
 	golang.org/x/net v0.0.0-20190311183353-d8887717615a // indirect
 	golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421 // indirect
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	k8s.io/api v0.0.0-20190222213804-5cb15d344471 // indirect
-	k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
-	k8s.io/client-go v2.0.0-alpha.0.0.20190228174230-b40b2a5939e4+incompatible
+	k8s.io/apimachinery v0.0.0-20190311155258-f9b45bc4494d
+	k8s.io/client-go v10.0.0+incompatible
```
Comment (guessi) [inline go.mod]: Here I pinned `k8s.io/apimachinery` to `86fb29eff628`
aim to pinned the version to the release version of `kubernetes-1.13.4`

```
% cd ${GOPATH}/src/k8s.io/apimachinery
% git show-ref --dereference kubernetes-1.13.4
adc9689aa021e10172b0df1c854261e7fd84e123 refs/tags/kubernetes-1.13.4
86fb29eff6288413d76bd8506874fddd9fccdff0 refs/tags/kubernetes-1.13.4^{}
```

but `f9b45bc4494d` is now pointed to the `master` of `k8s.io/apimachinery`
which is now under `beta` stage of `kubernetes-1.14.0`, or `alpha` stage of `kubernetes-1.15.0`

```
% git describe HEAD
kubernetes-1.14.0-beta.0-25-gf9b45bc
```

same for the `k8s.io/client-go`
`v10.0.0` is stand for `kubernetes-1.13.0` initial release
but I'd to pinned version at `kubernetes-1.13.4`, to align with other packages under `k8s.io`

```
% cd ${GOPATH}/src/k8s.io/client-go
% git diff v10.0.0...kubernetes-1.13.4 --stat
 .travis.yml                              |   5 ++---
 CHANGELOG.md                             | 108 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Godeps/Godeps.json                       | 170 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++--------------------------------------------------------------------------
 README.md                                |  25 +++++++++++-----------
 dynamic/fake/simple.go                   |   1 +
 rest/config.go                           |  26 ++++++++++++++---------
 rest/config_test.go                      |   1 +
 tools/clientcmd/client_config.go         |   7 ++++---
 tools/clientcmd/client_config_test.go    |  15 +------------
 transport/config.go                      |   7 ++++++-
 transport/round_trippers.go              |  39 +++++++++++++++++++++++++++++++---
 {rest => transport}/token_source.go      |  14 ++++++-------
 {rest => transport}/token_source_test.go |   2 +-
 13 files changed, 281 insertions(+), 139 deletions(-)
```

File: Makefile [hunk 3]
```diff
@@ -6,14 +6,14 @@ VERSION_PATCH  := 1
 VERSION_SUFFIX := -dev
 
 COMMIT  := $(shell git describe --always)
-PKGS    := $(shell go list ./... | grep -v /vendor)
+PKGS    := $(shell go list ./...)
```
Comment (guessi) [inline Makefile]: oops... thanks

File: Makefile (hunk 1 above)
Comment (appleboy) [reply Makefile]: > conflict with the "all" defined at line 53

done

File: go.mod [hunk 4]
```diff
@@ -16,18 +16,20 @@ require (
...
+	github.com/stretchr/testify v1.3.0 // indirect
 	go.opencensus.io v0.19.1 // indirect
 	golang.org/x/net v0.0.0-20190311183353-d8887717615a // indirect
 	golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421 // indirect
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
-	k8s.io/api v0.0.0-20190222213804-5cb15d344471 // indirect
-	k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
+	k8s.io/api v0.0.0-20190222213804-5cb15d344471
+	k8s.io/apimachinery v0.0.0-20190311155258-f9b45bc4494d
```
Comment (guessi) [inline go.mod]: `k8s.io/apimachinery` is still pointed to `master`, but it's okay for this case.
maybe I should add some test cases to ensure dependencies of `k8s.io/...` will not conflict.

File: Makefile (hunk 1 above)
Comment (guessi) [reply Makefile]: thanks

**Comment by guessi** [comment]
merge, and I will create ticket for TODOs later.

**Pull request merged**